uploader/src/main/resources/lambda-template.yaml

```diff
@@ -26,5 +26,5 @@
         PLUTO_TABLE_NAME: !Ref 'ManualPlutoTable'
     MemorySize: 512
     Role: !GetAtt LambdaExecutionRole.Arn
-    Runtime: "java8"
+    Runtime: "java11"
```

Comment:
Note that we should also have updated https://github.com/guardian/editorial-tools-platform/blob/main/cloudformation/media-service-account/media-atom-maker/media-atom-maker.yaml - not doing so led to `media-atom-expirer-error-alarm` & `media-atom-scheduler-error-alarm` going off:

![image](https://github.com/guardian/media-atom-maker/assets/52038/8ba57c08-0d08-4a00-a1b6-a72eb93f1fcd)

![image](https://github.com/guardian/media-atom-maker/assets/52038/8d3ce1c8-ad76-4ff6-981b-1be0ee91962d)

PR https://github.com/guardian/editorial-tools-platform/pull/760 fixes this - thank you Tom!